### PR TITLE
fix: Convert go-migrate error to something readable

### PR DESCRIPTION
**Pull request by disq**

### CODE REVIEW

File: migration/migrator/migrator.go [hunk 1]
```diff
@@ -13,7 +14,7 @@ import (
 	"github.com/cloudquery/cq-provider-sdk/provider/schema"
 
 	"github.com/golang-migrate/migrate/v4"
-	_ "github.com/golang-migrate/migrate/v4/database/postgres"
+	migratepostgres "github.com/golang-migrate/migrate/v4/database/postgres"
```
Comment (roneli) [inline migration/migrator/migrator.go:17]:
```suggestion
	mpg"github.com/golang-migrate/migrate/v4/database/postgres"
```

File: migration/migrator/migrator.go [hunk 2]
```diff
@@ -323,3 +323,18 @@ func dropTables(ctx context.Context, conn *pgx.Conn, table *schema.Table) error
...
+var ErrNoSchema = errors.New("CURRENT_SCHEMA seems empty, possibly due to empty search_path. Try `GRANT ALL PRIVILEGES ON <search_path> TO <user>`")
+
+func convertMigrateError(err error) error {
+	if err == nil {
+		return err
+	}
+
+	// https://github.com/golang-migrate/migrate/issues/696
+	if err == migratepostgres.ErrNoSchema || strings.Contains(err.Error(), `"current_schema": converting NULL to string`) {
+		return ErrNoSchema
```
Comment (roneli) [inline migration/migrator/migrator.go:336]: maybe return the error with the correct search path?

**Review by roneli — CHANGES_REQUESTED** [review]
Looks good, had some comments

File: migration/migrator/migrator.go (hunk 2 above)
Comment (disq) [reply migration/migrator/migrator.go:336]: hardcode `public`?

Comment (disq) [reply migration/migrator/migrator.go]: Done

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**